### DISCUSSION TITLE
Does Github clone repositories for backups?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#61160] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have a few `Python` projects, but I certainly don't consider them anything special. Anyone with a little knowledge of `Bash`, `Batch` or `Powershell`  can recreate my `Python` scripts within a few minutes. I have a script that prompts for user input, creates a folder, and moves a file to that folder. It's been cloned 14 times with 7 unique clones! I've never cloned my own repository, and this isn't my work account. I don't work in a field where having a GitHub account is beneficial or necessary,

I can't understand why that script would have so many clones, Does Github clone repositories locally for backups?

**Comment by christophercouch** [comment, marked as answer]
No, GitHub does not clone repositories for the purpose of local backups. However, GitHub does have its own backup and disaster recovery mechanisms in place to ensure data integrity. The clones you're seeing might be from other users who found your repository and decided to clone it, or from bots and crawlers that sometimes index public repositories.
